pkg/webapi/qrcode.go

```diff
@@ -1,12 +1,43 @@
 package webapi
 
 import (
+	"encoding/hex"
+	"fmt"
+	"image/color"
+
 	qrcode "github.com/skip2/go-qrcode"
 )
 
-func GenerateQRCodePNG(content string, size int) ([]byte, error) {
-	// Generate the QR code as a PNG image
-	pngBytes, err := qrcode.Encode(content, qrcode.Medium, size)
+func GenerateQRCodePNG(content string, size int, fg string, bg string) ([]byte, error) {
+	q, err := qrcode.New(content, qrcode.Medium)
+	q.ForegroundColor = color.White
+
+	DefaultFG := []byte{0, 0, 0, 254}
+	DefaultBG := []byte{255, 255, 255, 255}
+
+	// decode passed in fg/bg if sent
+	f, err := hex.DecodeString(fg)
```

Comment:
What happens if you pass the string "ff"

Reply:
fixed :)